lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -221,9 +230,13 @@ class Runner(
           forceLoggerConnectionArguments()
         else Seq()
 
-      val command = Seq(
-        javaCommand.executableName
-      ) ++ jvmArguments ++ loggingConnectionArguments ++ runSettings.runnerArguments
+      val command = Seq(javaCommand.executableName) ++
+        (if (runSettings.attachDebugger)
+           Seq(
+             "-agentlib:jdwp=transport=dt_socket,server=y,address=localhost:5005,suspend=y"
```

Comment:
Is this the _non trivial_ option we are make this for? I do understand your pain, I feel it too every time I have to type `transport=dt_socket`, but...